### ISSUE TITLE
rancher2_app_v2 values not passed to sub charts

### ISSUE THREAD

**Issue by davidtron** [rancher/terraform-provider-rancher2#545]
We have a private registry that we are using for an airgapped install.  
When we try to install rancher-monitoring using the terraform provider the systemDefaultRegistry is not passed through to the rancher-monitoring-crd subchart.

```
resource "rancher2_app_v2" "rancher-monitoring" {
  cluster_id = "blah"
  name = "rancher-monitoring"
  namespace = "cattle-monitoring-system"
  repo_name = "rancher-charts"
  chart_name = "rancher-monitoring"
  chart_version = "9.4.200"
  values = <<EOF
global:
  cattle:
    systemDefaultRegistry: "nexus3..."
EOF
}
```

We believe that this is due the fact that the provider only passes through [cattle, clusterId, and clusterName](https://github.com/rancher/terraform-provider-rancher2/blob/494be80135cd2f716b8b53ac194b0a59c14b00ae/rancher2/structure_app_v2.go#L68) to the subchart

**Comment by rawmind0** [comment]
Issue reproduced. Added a patch to fix it at PR #538

**Comment by Sartigan** [comment]
Hello,

I am encountering the same issue. I am using the provider `rancher/rancher2 v1.21.0`
